### PR TITLE
Upgrade

### PR DESCRIPTION
**Pull request by rajat-agarwal1**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
     ],
     "minimum-stability": "stable",
     "require": {
-        "php": ">=8.0",
```
Comment (rajeevrai) [inline composer.json:18]: why this change? we are not changing anything related to PHP version

**Review by rajeevrai — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (rajat-agarwal1) [reply composer.json:18]: Laravel 10 needs PHP 8.1 minimum, so making this change to allow PHP version 8.*

**Review by rajeevrai — APPROVED** [review]
